README.md

```diff
@@ -254,46 +254,34 @@ Examples, with a module named `porgy`:
 
 ```js
 // Disallow extensions on rc files:
-[
-  'package.json',
-  '.porgyrc',
-  'porgy.config.js'
-]
-
-// ESLint searches for configuration in these places:
-[
-  '.eslintrc.js',
+['package.json', '.porgyrc', 'porgy.config.js'][
+  // ESLint searches for configuration in these places:
+  ('.eslintrc.js',
```

Comment:
This formatting doesn't seem right.

Reply:
Prettier will format code inside markdown code blocks if it supports the language. If you end up wanting prettier on the readme / other markdown files, I'll want to update some of the examples so it is better understood as actual javascript.

Reply:
Markdown has been removed from the PR